sql/load_demo_data.sql

```diff
@@ -6100,7 +6291,11 @@ CREATE TABLE `oh_session_audit` (
   `SEA_ACTIVE` tinyint(1) NOT NULL DEFAULT 1,
```

Comment:
STEP 4:   
<pre>`SEA_ID` int(11) unsigned NOT NULL AUTO INCREMENT,</pre>
Demo:    
<pre>`SEA_ID` int(11) NOT NULL AUTO INCREMENT,</pre>

Reply:
Columns with `AUTO_INCREMENT` are unsigned by default, so the demo data (which is a dump) automatically omits it.
We can leave unchanged demo data and I'll just adapt step_84 that introduced the field.
Thanks!